Formula/icecast.rb

```diff
@@ -23,4 +24,8 @@ def install
     (prefix+"var/log/icecast").mkpath
     touch prefix+"var/log/icecast/error.log"
   end
+
+  test do
+    system "#{bin}/icecast", "-v"
```

Comment:
Thanks for adding a test, however:

We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
To reiterate, icecast is a program that sets up an internet radio broadcast which can be connected to by various music player software. It does not directly give any output to the console. I am not sure how to go about automatically testing this. Would checking log output be acceptable?

Reply:
Never mind, I found the tests that the dev team uses internally. I will translate to Ruby.

Reply:
I added the test, please re-review